doc/nrf/known_issues.rst

```diff
@@ -635,6 +635,13 @@ DESK-978: Directed advertising issues with SoftDevice Link Layer
 
   **Workaround:** Directed advertising is disabled by default for nRF Desktop.
 
+.. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-12020: Current for Gaming Mouse increased by 1400mA when not in sleep mode.
+  Gaming Mouse uses Zephyr driver for PWM_leds which sets pwm every cycle.
```

Comment:
```suggestion
  When not in the sleep mode, the Gaming Mouse reference design has current consumption higher by 1400mA.
  This is because this reference design uses Zephyr's driver for PWM_leds, which sets pwm every cycle.
```